content/en/docs/architecture/step-registry.md

```diff
@@ -175,6 +175,28 @@ ref:
 By default, only secrets in the `test-credentials` namespace will be available for mounting into test steps. Please follow the secret-mirroring
 [documentation](https://github.com/openshift/release/blob/master/core-services/secret-mirroring/README.md) to set up a custom secret in that namespace**
 
+##### Custom Bitwarden items
+
+Adding a new Bitwarden item to create a new secret or inject it in an existing one, requires few minor steps.
+First, the user has to create the Bitwarden item and share it with the *OpenShift TestPlatform (CI)* collection,
+following the official [documentation](https://bitwarden.com/help/article/share-to-a-collection/).
+
+After the item has been created and shared, it can be used in the [configuration file](https://github.com/openshift/release/blob/master/core-services/ci-secret-bootstrap/_config.yaml).
```

Comment:
I think that "it can be used in the configuration file" is *very* vague, to the point of being quite useless. The user does not want to use a secret in a configuration file, they want it avaialbe for consumption by a test step. So I think the text should go in that direction. Something like:

"After a secret is stored in Bitwarden, it needs to be made available on all clusters that execute CI jobs. For that, add a following stanza to the _configuration file_."